notebook/static/notebook/js/quickhelp.js

```diff
@@ -168,10 +184,8 @@ define([
         var element = $('<div/>');
 
         // The documentation
-        var doc = $('<div/>').addClass('alert alert-warning');
+        var doc = $('<div/>').addClass('alert alert-info');
         doc.append(
-            $('<button/>').addClass('close').attr('data-dismiss','alert').html('&times;')
-        ).append(
```

Comment:
This made no sens as there are no persistence of the hide this thing. So removed as well.